test/callstack_compression_test.ml

```diff
@@ -0,0 +1,82 @@
+open Core
+open Magic_trace_lib
+open Import
+
+let compressed_test_string =
+  "(\n\
```

Comment:
If you use raw strings (e.g. `{|raw string|}`), you won't need any backslashes in this test output.